src/thread/thread.cr

```diff
@@ -22,6 +22,14 @@ class Thread(T, R)
     LibPThread.detach(@th) unless @detached
   end
 
+  def alive?
+    if LibPThread.try_kill(@th, 0) == 0
```

Comment:
Well, it's not a try, it's the `0` that makes it a noop, I'd prefer keep calling it kill though.